fleece/log.py

```diff
@@ -112,6 +116,31 @@ def _has_streamhandler(logger, level=None, fmt=LOG_FORMAT,
     return False
 
 
+def inject_request_ids_into_environment(func):
+    """Decorator for the Lambda handler to inject request IDs for logging."""
+
+    @wraps(func)
+    def wrapper(event, context):
+        # This might not always be an API Gateway event, so only log the
+        # request ID, if it looks like to be coming from there.
+        if 'requestContext' in event:
+            os.environ[ENV_APIG_REQUEST_ID] = event['requestContext'].get(
+                'requestId', 'N/A')
+        os.environ[ENV_LAMBDA_REQUEST_ID] = context.aws_request_id
```

Comment:
I'm pretty sure it would, but since the request IDs are sort of related to the runtime environment, storing them there could be useful beyond the scope of this change at some point.